src/security/backups.md

```diff
@@ -0,0 +1,19 @@
+# Backup and restore
+
+Backups vary by our offering and their retention is governed by our [data retention](/security/data-retention.md) policies.
+
+Below are details on our Recovery Point Objective (RPO) and Recovery Time Objective (RTO) for our Platform Standard and Platform Enterprise offerings.
+
+## Platform Standard
+
+With Platform Standard, we have enabled our clients to manage their own backup and restore functions. Please see the [snapshot and restore](/administration/snapshot-and-restore.md) page for details.
+
+**RPO**: User defined. The RPO is configured by our clients.
+
+**RTO**: Variable. Recovery time will depend upon the size of the data we are recovering
+
+## Platform Enterprise
+
+**RPO**: 6 hours.  Platform.sh takes a backup of Platform.sh Enterprise environments every 6 hours.
```

Comment:
Is it worth noting here that we keep a sliding scale of backups, so this does not mean we have all old backups every 6 hours, just the recent ones?

Reply:
Interesting question. The answer would go on the retention page. Since this page is meant to answer compliance questions, and this line item is for PE, it might be better to discuss that as part of on-boarding vs put in the public docs (although I do have that information...it's just not easily consumable without some guidance).

Reply:
You can say "see retention policy" and link to it. Then I consider this question resolved.

Reply:
scratch that. You've linked to retention above.